2019-10-22 10:55:32.096 [info] <0.43.0> Application lager started on node nonode@nohost
2019-10-22 10:55:32.104 [error] <0.227.0> CRASH REPORT Process <0.227.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
2019-10-22 10:55:36.505 [info] <0.43.0> Application lager started on node nonode@nohost
2019-10-22 10:55:36.513 [error] <0.214.0> CRASH REPORT Process <0.214.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.Enumerable',value => nil} in 'Elixir.Enumerable':'impl_for!'/1 line 1
